gdal_subsetter/transform.py

```diff
@@ -126,20 +124,19 @@ def process_item(self, item, source):
         pystac.Item
             a STAC item containing the Zarr output
         """
-        logger = self.logger
-        message = self.message
-
-        if message.subset and message.subset.shape:
-            logger.warn('Ignoring subset request for user shapefile %s' %
-                        (message.subset.shape.href,))
+        if self.message.subset and self.message.subset.shape:
+            self.logger.warning('Ignoring subset request for user shapefile '
+                                f'{self.message.subset.shape.href}')
 
         layernames = []
-        operations = dict(
-            variable_subset=source.variables,
-            is_regridded=bool(message.format.crs or message.format.width or message.format.height),
-            is_subsetted=bool(message.subset and (message.subset.bbox or message.subset.shape) )
-        )
-        
+        operations = {'variable_subset': source.variables,
+                      'is_regridded': bool(self.message.format.crs
+                                           or self.message.format.width
+                                           or self.message.format.height),
+                      'is_subsetted': bool(self.message.subset
+                                           and (self.message.subset.bbox
+                                                or self.message.subset.shape))}
+
```

Comment:
Is it now standard practice to not use "dict"? (We talked about this format during your review of my Jupyter notebook).

Reply:
This is an interesting question. My personal preference is to use the standard Python nomenclature of `{'key': 'value'}` to instantiate a dictionary, although the `dict(key='value')` constructor function achieves the same thing. If you look at the [Python documentation](https://docs.python.org/3/tutorial/datastructures.html#dictionaries), the majority of examples use the standard nomenclature, rather than the `dict()` constructor.

I took went a little way down the rabbit hole, and took a quick look at the relative benefits, and it looks like the main difference is that using the expected nomenclature of `{'key': 'value'}` does actually [offer slightly better performance](http://katrin-affolter.ch/Python/Python_dictionaries) (although at this size of a dictionary, that's really not significant).